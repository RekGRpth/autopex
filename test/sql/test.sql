CREATE EXTENSION autopex;

CREATE EXTENSION plsh;

DO $$#!/bin/sh
true$$ LANGUAGE plsh;
