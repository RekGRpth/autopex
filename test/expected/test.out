CREATE EXTENSION autopex;
CREATE EXTENSION plsh;
NOTICE:  autopex: downloading and building extension; this might take a moment
DO $$#!/bin/sh
true$$ LANGUAGE plsh;
